client/src/main/java/com/paypal/selion/platform/grid/SeLionAppiumIOSDriver.java

```diff
@@ -76,64 +64,71 @@ public WebElement findElementBy(By by) {
     @Override
     public void doubleTap(WebElement webElement) {
         logger.entering(webElement);
-        String elementId = ((RemoteWebElement) webElement).getId();
-        Map<String, String> optionsMap = createOptionsForDoubleTap(elementId, ONE_FINGER);
-        javaScriptExecutor.executeScript(TAP_SCRIPT, optionsMap);
+        new TouchAction(this)
+                .tap(webElement).release()
+                .waitAction(100)
```

Comment:
What makes `100` and acceptable value here? It seems awfully like `magic number` in this context. Can we fix this?